openvino/CMakeLists.txt

```diff
@@ -15,5 +15,7 @@
 # ******************************************************************************
 
 add_subdirectory(itt)
+add_subdirectory(cc)
```

Comment:
can we have better name? I don't mind about `conditional_compilation`

Reply:
I agree that we need a better name

Reply:
fixed